src/components/Filters.js

```diff
@@ -0,0 +1,64 @@
+
+const Filters = props => {
+
+    const displayFruit = () => {
```

Comment:
so currently you can't switch from Fruit to Veg filters without resetting since you're removing items from the store and have no way of getting them back. One way around this would be to have the filter function modify rendering, not state. So the filter would change what renders (only fruits, or only veg) - but the state stays the same.

Reply:
ahhh! you have the best feedbacks thankyou!!